### DISCUSSION TITLE
Moes Thermostat ZHT-GA-006-WH-MG supported ?

### DISCUSSION THREAD

**Discussion by guny74** [Koenkk/zigbee2mqtt#16287] [Devices]
Hello, 

I did not finde the following device in the supported list, but from the features it looks similar to the Moes BHT-002  

May it be nevertheless supported ?

https://moeshouse.com/collections/smart-thermostat/products/zigbee-smart-thermostat-gas-water-heater-room-thermostat-digital-programmable  (here the GC Version) 

Thank you

**Comment by elemarek** [comment]
I have this thermostat model under the name BHT-006GAZB https://www.aliexpress.com/item/1005004829125369.html?spm=a2g0o.order_list.order_list_main.11.21ef1802oIP0WN. Zigbee2mqtt recognizes it as BHT-002-GCLZB. Not all functions work.

**Reply by guny74** [reply]
Ok, it arrived and it is connected.... and it is true not all function works, but I hoped that little bit more function would work...
so I could not use it.... I will open an request...

**Comment by guny74** [comment]
thank you. I buy and try it....

regards

**Comment by sweti3** [comment]
Hei is there any news to this here? Does it work in zigbee2mqtt now and if not so, what could we do about that? Thanks so much!

**Reply by Kees1957** [reply]
I have the same issues and it looks like TYBAC-006 is the same thermostat (in the devices list), but then from Tuya. I am a newbee and have no idea how to change to the TYBAC-006 config. Anyone?

**Comment by rego82** [comment]
I've also included it in HASS. however, the BHT-002 is still recognized. Unfortunately, the processor load of the raspi 4 increases by 5% and it crashes regularly. As soon as I disconnect the device, the problems disappear.
Is there any  chance for a clean integration?
